src/commands/prepare.ts

```diff
@@ -384,6 +391,14 @@ async function prepareChangelog(
       }
       if (!changeset.body) {
         replaceSection = changeset.name;
+        const changes = await getChangesSince(git, oldVersion);
+        changeset.body = changes
+          .map(change =>
+            change.pr
+              ? `- ${change.message}`
+              : `- ${change.message} (${change.hash.slice(0, 8)})`
```

Comment:
Any reason to set 8?

Reply:
That's the common length for short SHAs on GitHub 🙂

Reply:
Eh, you are right about getting confused. Git defaults to `7` but this is known to cause collisions with larger projects (such as Sentry) so AFAIK the default for GitHub is 8+ based on some StackOverflow research: https://stackoverflow.com/questions/18134627/how-much-of-a-git-sha-is-generally-considered-necessary-to-uniquely-identify-a

Will add a comment in a follow-up.

Reply:
> Will add a comment in a follow-up.

Bump ☝️ 

Also, I think the pr branch logic would've been cleaner to encapsulate in `getChangeSince` since the PR extraction logic is also down there, and what we're basically doing here is defaulting to a SHA in the case that we can't find a PR.